sdc/src/sdc/main.d

```diff
@@ -29,17 +29,33 @@ int main(string[] args) {
 	uint optLevel;
 	bool dontLink;
 	string outputFile;
-	getopt(
+
+	auto help_info = getopt(
 		args, std.getopt.config.caseSensitive,
-		"I", &includePath,
-		"O", &optLevel,
-		"c", &dontLink,
-		"o", &outputFile,
-		"help|h", delegate() {
-			import std.stdio;
-			writeln("HELP !");
-		}
+		"include|I",    "Include path",        &includePath,
+		"optlevel|O",   "Optimization level",  &optLevel,
+		"nolink|c",     "Stop before linking", &dontLink,
+		"output|o",     "Output file",         &outputFile
 	);
+
+    if (help_info.helpWanted)
```

Comment:
Got to do that at some point. But there is so much stuff higher in the TODO list...